README.md

```diff
@@ -19,16 +19,14 @@ This parser is in the ***experimental stages*** of development.
 
 At least it works fine with a [fork of the `astro.build` repository](https://github.com/ota-meshi/astro.build/tree/eslint).
 
-⚠ Currently this parser relies heavily on the internal API of [@astrojs/compiler]. It may stop working in a future update of [@astrojs/compiler]. ⚠
```

Comment:
@natemoo-re has been working on `.astro -> .tsx` output from the Astro compiler for our VSCode extension, is this something that you would be able to use in your parser?

Reply:
Hmm. I find it difficult to use TSX with this ESLint parser because the ESLint parser requires an AST and a token array with the correct location. For example, I don't think we can know the position of the `---` fence token from TSX.

Reply:
What if the compiler output included a source map? Would you be able to parse the tsx and then use the map to point back to the original locations?

Reply:
As you say, script and HTML ASTs can probably get location by remapping using a source map. However, since TSX does not include a frontmatter fence, I don't think the location of the fence will be known from AST.
Also, the remapping needs to be able to get the exact location of each token. Otherwise, ESLint auto-fix will break the source code.

Having said that, It may be worth a try :smile:

Reply:
Ah, right! That makes sense.

Reply:
I think this all makes sense! The compiler has an experimental `convertToTSX` function now which returns `{ code, map }`. Not sure it will be _more_ useful than `parse`, though.